### PR TITLE
fix(components): [pagination] style

### LINKED ISSUE #10555: [Style] [pagination] 分页组件布局问题
Bug Type: **`Style`**

## Environment

- Vue Version: `3.2.44`
- Element Plus Version: `2.2.20`
- Browser / OS: `Chrome/107.0.0.0 Safari/537.36 Edg/107.0.1418.35`
- Build Tool: `Vite`

### Related Component

- `el-pagination`

### Reproduction Link

[Element Plus Playground](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZGl2IGNsYXNzPVwiZmxleCBpdGVtcy1jZW50ZXIgbWItNFwiPlxuICAgIDxlbC1yYWRpby1ncm91cCB2LW1vZGVsPVwic21hbGxcIiBjbGFzcz1cIm1yLTRcIj5cbiAgICAgIDxlbC1yYWRpby1idXR0b24gOmxhYmVsPVwiZmFsc2VcIj5kZWZhdWx0PC9lbC1yYWRpby1idXR0b24+XG4gICAgICA8ZWwtcmFkaW8tYnV0dG9uIDpsYWJlbD1cInRydWVcIj5zbWFsbDwvZWwtcmFkaW8tYnV0dG9uPlxuICAgIDwvZWwtcmFkaW8tZ3JvdXA+XG4gICAgPGRpdj5cbiAgICAgIGJhY2tncm91bmQ6XG4gICAgICA8ZWwtc3dpdGNoIHYtbW9kZWw9XCJiYWNrZ3JvdW5kXCIgY2xhc3M9XCJtbC0yXCIgLz5cbiAgICA8L2Rpdj5cbiAgICA8ZGl2IGNsYXNzPVwibWwtNFwiPlxuICAgICAgZGlzYWJsZWQ6IDxlbC1zd2l0Y2ggdi1tb2RlbD1cImRpc2FibGVkXCIgY2xhc3M9XCJtbC0yXCIgLz5cbiAgICA8L2Rpdj5cbiAgPC9kaXY+XG5cbiAgPGhyIGNsYXNzPVwibXktNFwiIC8+XG5cbiAgPGRpdiBjbGFzcz1cImRlbW8tcGFnaW5hdGlvbi1ibG9ja1wiPlxuICAgIDxkaXYgY2xhc3M9XCJkZW1vbnN0cmF0aW9uXCI+VG90YWwgaXRlbSBjb3VudDwvZGl2PlxuICAgIDxlbC1wYWdpbmF0aW9uXG4gICAgICB2LW1vZGVsOmN1cnJlbnRQYWdlPVwiY3VycmVudFBhZ2UxXCJcbiAgICAgIDpwYWdlLXNpemU9XCIxMDBcIlxuICAgICAgOnNtYWxsPVwic21hbGxcIlxuICAgICAgOmRpc2FibGVkPVwiZGlzYWJsZWRcIlxuICAgICAgOmJhY2tncm91bmQ9XCJiYWNrZ3JvdW5kXCJcbiAgICAgIGxheW91dD1cInRvdGFsLCBwcmV2LCBwYWdlciwgbmV4dFwiXG4gICAgICA6dG90YWw9XCIxMDAwXCJcbiAgICAgIEBzaXplLWNoYW5nZT1cImhhbmRsZVNpemVDaGFuZ2VcIlxuICAgICAgQGN1cnJlbnQtY2hhbmdlPVwiaGFuZGxlQ3VycmVudENoYW5nZVwiXG4gICAgLz5cbiAgPC9kaXY+XG4gIDxkaXYgY2xhc3M9XCJkZW1vLXBhZ2luYXRpb24tYmxvY2tcIj5cbiAgICA8ZGl2IGNsYXNzPVwiZGVtb25zdHJhdGlvblwiPkNoYW5nZSBwYWdlIHNpemU8L2Rpdj5cbiAgICA8ZWwtcGFnaW5hdGlvblxuICAgICAgdi1tb2RlbDpjdXJyZW50UGFnZT1cImN1cnJlbnRQYWdlMlwiXG4gICAgICB2LW1vZGVsOnBhZ2Utc2l6ZT1cInBhZ2VTaXplMlwiXG4gICAgICA6cGFnZS1zaXplcz1cIlsxMDAsIDIwMCwgMzAwLCA0MDBdXCJcbiAgICAgIDpzbWFsbD1cInNtYWxsXCJcbiAgICAgIDpkaXNhYmxlZD1cImRpc2FibGVkXCJcbiAgICAgIDpiYWNrZ3JvdW5kPVwiYmFja2dyb3VuZFwiXG4gICAgICBsYXlvdXQ9XCIgcHJldiwgcGFnZXIsIG5leHQsc2l6ZXMsdG90YWxcIlxuICAgICAgOnRvdGFsPVwiMTAwMFwiXG4gICAgICBAc2l6ZS1jaGFuZ2U9XCJoYW5kbGVTaXplQ2hhbmdlXCJcbiAgICAgIEBjdXJyZW50LWNoYW5nZT1cImhhbmRsZUN1cnJlbnRDaGFuZ2VcIlxuICAgIC8+XG4gIDwvZGl2PlxuICA8ZGl2IGNsYXNzPVwiZGVtby1wYWdpbmF0aW9uLWJsb2NrXCI+XG4gICAgPGRpdiBjbGFzcz1cImRlbW9uc3RyYXRpb25cIj5KdW1wIHRvPC9kaXY+XG4gICAgPGVsLXBhZ2luYXRpb25cbiAgICAgIHYtbW9kZWw6Y3VycmVudFBhZ2U9XCJjdXJyZW50UGFnZTNcIlxuICAgICAgdi1tb2RlbDpwYWdlLXNpemU9XCJwYWdlU2l6ZTNcIlxuICAgICAgOnNtYWxsPVwic21hbGxcIlxuICAgICAgOmRpc2FibGVkPVwiZGlzYWJsZWRcIlxuICAgICAgOmJhY2tncm91bmQ9XCJiYWNrZ3JvdW5kXCJcbiAgICAgIGxheW91dD1cInByZXYsIHBhZ2VyLCBuZXh0LCBqdW1wZXJcIlxuICAgICAgOnRvdGFsPVwiMTAwMFwiXG4gICAgICBAc2l6ZS1jaGFuZ2U9XCJoYW5kbGVTaXplQ2hhbmdlXCJcbiAgICAgIEBjdXJyZW50LWNoYW5nZT1cImhhbmRsZUN1cnJlbnRDaGFuZ2VcIlxuICAgIC8+XG4gIDwvZGl2PlxuICA8ZGl2IGNsYXNzPVwiZGVtby1wYWdpbmF0aW9uLWJsb2NrXCI+XG4gICAgPGRpdiBjbGFzcz1cImRlbW9uc3RyYXRpb25cIj5BbGwgY29tYmluZWQ8L2Rpdj5cbiAgICA8ZWwtcGFnaW5hdGlvblxuICAgICAgdi1tb2RlbDpjdXJyZW50UGFnZT1cImN1cnJlbnRQYWdlNFwiXG4gICAgICB2LW1vZGVsOnBhZ2Utc2l6ZT1cInBhZ2VTaXplNFwiXG4gICAgICA6cGFnZS1zaXplcz1cIlsxMDAsIDIwMCwgMzAwLCA0MDBdXCJcbiAgICAgIDpzbWFsbD1cInNtYWxsXCJcbiAgICAgIDpkaXNhYmxlZD1cImRpc2FibGVkXCJcbiAgICAgIDpiYWNrZ3JvdW5kPVwiYmFja2dyb3VuZFwiXG4gICAgICBsYXlvdXQ9XCJ0b3RhbCwgc2l6ZXMsIHByZXYsIHBhZ2VyLCBuZXh0LCBqdW1wZXJcIlxuICAgICAgOnRvdGFsPVwiNDAwXCJcbiAgICAgIEBzaXplLWNoYW5nZT1cImhhbmRsZVNpemVDaGFuZ2VcIlxuICAgICAgQGN1cnJlbnQtY2hhbmdlPVwiaGFuZGxlQ3VycmVudENoYW5nZVwiXG4gICAgLz5cbiAgPC9kaXY+XG48L3RlbXBsYXRlPlxuPHNjcmlwdCBsYW5nPVwidHNcIiBzZXR1cD5cbmltcG9ydCB7IHJlZiB9IGZyb20gJ3Z1ZSdcblxuY29uc3QgY3VycmVudFBhZ2UxID0gcmVmKDUpXG5jb25zdCBjdXJyZW50UGFnZTIgPSByZWYoNSlcbmNvbnN0IGN1cnJlbnRQYWdlMyA9IHJlZig1KVxuY29uc3QgY3VycmVudFBhZ2U0ID0gcmVmKDQpXG5jb25zdCBwYWdlU2l6ZTIgPSByZWYoMTAwKVxuY29uc3QgcGFnZVNpemUzID0gcmVmKDEwMClcbmNvbnN0IHBhZ2VTaXplNCA9IHJlZigxMDApXG5jb25zdCBzbWFsbCA9IHJlZihmYWxzZSlcbmNvbnN0IGJhY2tncm91bmQgPSByZWYoZmFsc2UpXG5jb25zdCBkaXNhYmxlZCA9IHJlZihmYWxzZSlcblxuY29uc3QgaGFuZGxlU2l6ZUNoYW5nZSA9ICh2YWw6IG51bWJlcikgPT4ge1xuICBjb25zb2xlLmxvZyhgJHt2YWx9IGl0ZW1zIHBlciBwYWdlYClcbn1cbmNvbnN0IGhhbmRsZUN1cnJlbnRDaGFuZ2UgPSAodmFsOiBudW1iZXIpID0+IHtcbiAgY29uc29sZS5sb2coYGN1cnJlbnQgcGFnZTogJHt2YWx9YClcbn1cbjwvc2NyaXB0PlxuXG48c3R5bGUgc2NvcGVkPlxuLmRlbW8tcGFnaW5hdGlvbi1ibG9jayArIC5kZW1vLXBhZ2luYXRpb24tYmxvY2sge1xuICBtYXJnaW4tdG9wOiAxMHB4O1xufVxuLmRlbW8tcGFnaW5hdGlvbi1ibG9jayAuZGVtb25zdHJhdGlvbiB7XG4gIG1hcmdpbi1ib3R0b206IDE2cHg7XG59XG48L3N0eWxlPlxuIiwiaW1wb3J0X21hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge31cbn0iLCJfbyI6e319)

### Steps to reproduce

带背景分页  布局为 prev, pager, next, sizes, total

### What is Expected?

修复样式问题

### What is actually happening?

下一页与size选择器没有间隔

### Additional comments

_(empty)_

### PR DESCRIPTION
**Pull request by Simon-He95**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

close: #10555

### CODE REVIEW

**Comment by Simon-He95** [comment]
[playground](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZGl2IGNsYXNzPVwiZmxleCBpdGVtcy1jZW50ZXIgbWItNFwiPlxuICAgIDxlbC1yYWRpby1ncm91cCB2LW1vZGVsPVwic21hbGxcIiBjbGFzcz1cIm1yLTRcIj5cbiAgICAgIDxlbC1yYWRpby1idXR0b24gOmxhYmVsPVwiZmFsc2VcIj5kZWZhdWx0PC9lbC1yYWRpby1idXR0b24+XG4gICAgICA8ZWwtcmFkaW8tYnV0dG9uIDpsYWJlbD1cInRydWVcIj5zbWFsbDwvZWwtcmFkaW8tYnV0dG9uPlxuICAgIDwvZWwtcmFkaW8tZ3JvdXA+XG4gICAgPGRpdj5cbiAgICAgIGJhY2tncm91bmQ6XG4gICAgICA8ZWwtc3dpdGNoIHYtbW9kZWw9XCJiYWNrZ3JvdW5kXCIgY2xhc3M9XCJtbC0yXCIgLz5cbiAgICA8L2Rpdj5cbiAgICA8ZGl2IGNsYXNzPVwibWwtNFwiPlxuICAgICAgZGlzYWJsZWQ6IDxlbC1zd2l0Y2ggdi1tb2RlbD1cImRpc2FibGVkXCIgY2xhc3M9XCJtbC0yXCIgLz5cbiAgICA8L2Rpdj5cbiAgPC9kaXY+XG5cbiAgPGhyIGNsYXNzPVwibXktNFwiIC8+XG5cbiAgPGRpdiBjbGFzcz1cImRlbW8tcGFnaW5hdGlvbi1ibG9ja1wiPlxuICAgIDxkaXYgY2xhc3M9XCJkZW1vbnN0cmF0aW9uXCI+VG90YWwgaXRlbSBjb3VudDwvZGl2PlxuICAgIDxlbC1wYWdpbmF0aW9uXG4gICAgICB2LW1vZGVsOmN1cnJlbnRQYWdlPVwiY3VycmVudFBhZ2UxXCJcbiAgICAgIDpwYWdlLXNpemU9XCIxMDBcIlxuICAgICAgOnNtYWxsPVwic21hbGxcIlxuICAgICAgOmRpc2FibGVkPVwiZGlzYWJsZWRcIlxuICAgICAgOmJhY2tncm91bmQ9XCJiYWNrZ3JvdW5kXCJcbiAgICAgIGxheW91dD1cInRvdGFsLCBwcmV2LCBwYWdlciwgbmV4dFwiXG4gICAgICA6dG90YWw9XCIxMDAwXCJcbiAgICAgIEBzaXplLWNoYW5nZT1cImhhbmRsZVNpemVDaGFuZ2VcIlxuICAgICAgQGN1cnJlbnQtY2hhbmdlPVwiaGFuZGxlQ3VycmVudENoYW5nZVwiXG4gICAgLz5cbiAgPC9kaXY+XG4gIDxkaXYgY2xhc3M9XCJkZW1vLXBhZ2luYXRpb24tYmxvY2tcIj5cbiAgICA8ZGl2IGNsYXNzPVwiZGVtb25zdHJhdGlvblwiPkNoYW5nZSBwYWdlIHNpemU8L2Rpdj5cbiAgICA8ZWwtcGFnaW5hdGlvblxuICAgICAgdi1tb2RlbDpjdXJyZW50UGFnZT1cImN1cnJlbnRQYWdlMlwiXG4gICAgICB2LW1vZGVsOnBhZ2Utc2l6ZT1cInBhZ2VTaXplMlwiXG4gICAgICA6cGFnZS1zaXplcz1cIlsxMDAsIDIwMCwgMzAwLCA0MDBdXCJcbiAgICAgIDpzbWFsbD1cInNtYWxsXCJcbiAgICAgIDpkaXNhYmxlZD1cImRpc2FibGVkXCJcbiAgICAgIDpiYWNrZ3JvdW5kPVwiYmFja2dyb3VuZFwiXG4gICAgICBsYXlvdXQ9XCIgcHJldiwgcGFnZXIsIG5leHQsc2l6ZXMsdG90YWxcIlxuICAgICAgOnRvdGFsPVwiMTAwMFwiXG4gICAgICBAc2l6ZS1jaGFuZ2U9XCJoYW5kbGVTaXplQ2hhbmdlXCJcbiAgICAgIEBjdXJyZW50LWNoYW5nZT1cImhhbmRsZUN1cnJlbnRDaGFuZ2VcIlxuICAgIC8+XG4gIDwvZGl2PlxuICA8ZGl2IGNsYXNzPVwiZGVtby1wYWdpbmF0aW9uLWJsb2NrXCI+XG4gICAgPGRpdiBjbGFzcz1cImRlbW9uc3RyYXRpb25cIj5KdW1wIHRvPC9kaXY+XG4gICAgPGVsLXBhZ2luYXRpb25cbiAgICAgIHYtbW9kZWw6Y3VycmVudFBhZ2U9XCJjdXJyZW50UGFnZTNcIlxuICAgICAgdi1tb2RlbDpwYWdlLXNpemU9XCJwYWdlU2l6ZTNcIlxuICAgICAgOnNtYWxsPVwic21hbGxcIlxuICAgICAgOmRpc2FibGVkPVwiZGlzYWJsZWRcIlxuICAgICAgOmJhY2tncm91bmQ9XCJiYWNrZ3JvdW5kXCJcbiAgICAgIGxheW91dD1cInByZXYsIHBhZ2VyLCBuZXh0LCBqdW1wZXJcIlxuICAgICAgOnRvdGFsPVwiMTAwMFwiXG4gICAgICBAc2l6ZS1jaGFuZ2U9XCJoYW5kbGVTaXplQ2hhbmdlXCJcbiAgICAgIEBjdXJyZW50LWNoYW5nZT1cImhhbmRsZUN1cnJlbnRDaGFuZ2VcIlxuICAgIC8+XG4gIDwvZGl2PlxuICA8ZGl2IGNsYXNzPVwiZGVtby1wYWdpbmF0aW9uLWJsb2NrXCI+XG4gICAgPGRpdiBjbGFzcz1cImRlbW9uc3RyYXRpb25cIj5BbGwgY29tYmluZWQ8L2Rpdj5cbiAgICA8ZWwtcGFnaW5hdGlvblxuICAgICAgdi1tb2RlbDpjdXJyZW50UGFnZT1cImN1cnJlbnRQYWdlNFwiXG4gICAgICB2LW1vZGVsOnBhZ2Utc2l6ZT1cInBhZ2VTaXplNFwiXG4gICAgICA6cGFnZS1zaXplcz1cIlsxMDAsIDIwMCwgMzAwLCA0MDBdXCJcbiAgICAgIDpzbWFsbD1cInNtYWxsXCJcbiAgICAgIDpkaXNhYmxlZD1cImRpc2FibGVkXCJcbiAgICAgIDpiYWNrZ3JvdW5kPVwiYmFja2dyb3VuZFwiXG4gICAgICBsYXlvdXQ9XCJ0b3RhbCwgc2l6ZXMsIHByZXYsIHBhZ2VyLCBuZXh0LCBqdW1wZXJcIlxuICAgICAgOnRvdGFsPVwiNDAwXCJcbiAgICAgIEBzaXplLWNoYW5nZT1cImhhbmRsZVNpemVDaGFuZ2VcIlxuICAgICAgQGN1cnJlbnQtY2hhbmdlPVwiaGFuZGxlQ3VycmVudENoYW5nZVwiXG4gICAgLz5cbiAgPC9kaXY+XG48L3RlbXBsYXRlPlxuPHNjcmlwdCBsYW5nPVwidHNcIiBzZXR1cD5cbmltcG9ydCB7IHJlZiB9IGZyb20gJ3Z1ZSdcblxuY29uc3QgY3VycmVudFBhZ2UxID0gcmVmKDUpXG5jb25zdCBjdXJyZW50UGFnZTIgPSByZWYoNSlcbmNvbnN0IGN1cnJlbnRQYWdlMyA9IHJlZig1KVxuY29uc3QgY3VycmVudFBhZ2U0ID0gcmVmKDQpXG5jb25zdCBwYWdlU2l6ZTIgPSByZWYoMTAwKVxuY29uc3QgcGFnZVNpemUzID0gcmVmKDEwMClcbmNvbnN0IHBhZ2VTaXplNCA9IHJlZigxMDApXG5jb25zdCBzbWFsbCA9IHJlZihmYWxzZSlcbmNvbnN0IGJhY2tncm91bmQgPSByZWYoZmFsc2UpXG5jb25zdCBkaXNhYmxlZCA9IHJlZihmYWxzZSlcblxuY29uc3QgaGFuZGxlU2l6ZUNoYW5nZSA9ICh2YWw6IG51bWJlcikgPT4ge1xuICBjb25zb2xlLmxvZyhgJHt2YWx9IGl0ZW1zIHBlciBwYWdlYClcbn1cbmNvbnN0IGhhbmRsZUN1cnJlbnRDaGFuZ2UgPSAodmFsOiBudW1iZXIpID0+IHtcbiAgY29uc29sZS5sb2coYGN1cnJlbnQgcGFnZTogJHt2YWx9YClcbn1cbjwvc2NyaXB0PlxuXG48c3R5bGUgc2NvcGVkPlxuLmRlbW8tcGFnaW5hdGlvbi1ibG9jayArIC5kZW1vLXBhZ2luYXRpb24tYmxvY2sge1xuICBtYXJnaW4tdG9wOiAxMHB4O1xufVxuLmRlbW8tcGFnaW5hdGlvbi1ibG9jayAuZGVtb25zdHJhdGlvbiB7XG4gIG1hcmdpbi1ib3R0b206IDE2cHg7XG59XG48L3N0eWxlPlxuIiwiUGxheWdyb3VuZE1haW4udnVlIjoiPHNjcmlwdCBzZXR1cD5cbmltcG9ydCBBcHAgZnJvbSAnLi9BcHAudnVlJ1xuaW1wb3J0IHsgc2V0dXBFbGVtZW50UGx1cyB9IGZyb20gJy4vZWxlbWVudC1wbHVzLmpzJ1xuc2V0dXBFbGVtZW50UGx1cygpXG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8QXBwIC8+XG48L3RlbXBsYXRlPlxuIiwiaW1wb3J0X21hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge1xuICAgIFwiZWxlbWVudC1wbHVzXCI6IFwiaHR0cHM6Ly9wcmV2aWV3LTEwNTU2LWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguZnVsbC5taW4ubWpzXCIsXG4gICAgXCJlbGVtZW50LXBsdXMvXCI6IFwidW5zdXBwb3J0ZWRcIlxuICB9XG59IiwiaW1wb3J0LW1hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge1xuICAgIFwidnVlXCI6IFwiaHR0cHM6Ly9mYXN0bHkuanNkZWxpdnIubmV0L25wbS9AdnVlL3J1bnRpbWUtZG9tQGxhdGVzdC9kaXN0L3J1bnRpbWUtZG9tLmVzbS1icm93c2VyLmpzXCIsXG4gICAgXCJAdnVlL3NoYXJlZFwiOiBcImh0dHBzOi8vZmFzdGx5LmpzZGVsaXZyLm5ldC9ucG0vQHZ1ZS9zaGFyZWRAbGF0ZXN0L2Rpc3Qvc2hhcmVkLmVzbS1idW5kbGVyLmpzXCIsXG4gICAgXCJlbGVtZW50LXBsdXNcIjogXCJodHRwczovL3ByZXZpZXctMTA1NTYtZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5mdWxsLm1pbi5tanNcIixcbiAgICBcImVsZW1lbnQtcGx1cy9cIjogXCJ1bnN1cHBvcnRlZFwiLFxuICAgIFwiQGVsZW1lbnQtcGx1cy9pY29ucy12dWVcIjogXCJodHRwczovL2Zhc3RseS5qc2RlbGl2ci5uZXQvbnBtL0BlbGVtZW50LXBsdXMvaWNvbnMtdnVlQDIvZGlzdC9pbmRleC5taW4uanNcIlxuICB9LFxuICBcInNjb3Blc1wiOiB7fVxufSIsImVsZW1lbnQtcGx1cy5qcyI6ImltcG9ydCB7IGdldEN1cnJlbnRJbnN0YW5jZSB9IGZyb20gJ3Z1ZSdcbmltcG9ydCBFbGVtZW50UGx1cyBmcm9tICdlbGVtZW50LXBsdXMnXG5cbmxldCBpbnN0YWxsZWQgPSBmYWxzZVxuYXdhaXQgbG9hZFN0eWxlKClcblxuZXhwb3J0IGZ1bmN0aW9uIHNldHVwRWxlbWVudFBsdXMoKSB7XG4gIGlmIChpbnN0YWxsZWQpIHJldHVyblxuICBjb25zdCBpbnN0YW5jZSA9IGdldEN1cnJlbnRJbnN0YW5jZSgpXG4gIGluc3RhbmNlLmFwcENvbnRleHQuYXBwLnVzZShFbGVtZW50UGx1cylcbiAgaW5zdGFsbGVkID0gdHJ1ZVxufVxuXG5leHBvcnQgZnVuY3Rpb24gbG9hZFN0eWxlKCkge1xuICByZXR1cm4gbmV3IFByb21pc2UoKHJlc29sdmUsIHJlamVjdCkgPT4ge1xuICAgIGNvbnN0IGxpbmsgPSBkb2N1bWVudC5jcmVhdGVFbGVtZW50KCdsaW5rJylcbiAgICBsaW5rLnJlbCA9ICdzdHlsZXNoZWV0J1xuICAgIGxpbmsuaHJlZiA9ICdodHRwczovL3ByZXZpZXctMTA1NTYtZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5jc3MnXG4gICAgbGluay5hZGRFdmVudExpc3RlbmVyKCdsb2FkJywgcmVzb2x2ZSlcbiAgICBsaW5rLmFkZEV2ZW50TGlzdGVuZXIoJ2Vycm9yJywgcmVqZWN0KVxuICAgIGRvY3VtZW50LmJvZHkuYXBwZW5kKGxpbmspXG4gIH0pXG59IiwiX28iOnsic2hvd0hpZGRlbiI6dHJ1ZSwic3R5bGVTb3VyY2UiOiJodHRwczovL3ByZXZpZXctMTA1NTYtZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5jc3MifX0=)

**Comment by kooriookami** [comment]
image
image
image
One more gap.

**Comment by Simon-He95** [comment]
fixed

**Comment by kooriookami** [comment]
> fixed

I found this change may cause other problem, and I will refactor this style later~

File: packages/theme-chalk/src/pagination.scss [hunk 1]
```diff
@@ -77,6 +77,9 @@
       background-color: getCssVar('pagination-button-disabled-bg-color');
       cursor: not-allowed;
     }
+    & +.#{$namespace}-pagination__sizes{
```
Comment (ryuhangyeong) [inline packages/theme-chalk/src/pagination.scss:80]: ## Suggest

minor opinion, but some code should be cleaned up for consistency with others.

```css
& + .#{$namespace}-pagination__sizes { // here

}
```

```css
margin-left: 16px; // here
```

also, I think the 177 to 178 line code is unnecessary.

**Comment by chenxch** [comment]
via #10563

**Pull request closed without merging**